task/buildah/0.1/buildah.yaml

```diff
@@ -100,10 +100,10 @@ spec:
     - name: COMMIT_SHA
       value: $(params.COMMIT_SHA)
     script: |
-      if [ -e "$CONTEXT/$DOCKERFILE" ]; then
-        dockerfile_path="$CONTEXT/$DOCKERFILE"
-      elif [ -e "$DOCKERFILE" ]; then
-        dockerfile_path="$DOCKERFILE"
+      if [ -e "source/$CONTEXT/$DOCKERFILE" ]; then
+        dockerfile_path="source/$CONTEXT/$DOCKERFILE"
```

Comment:
Seeing multiple places in this file `source` is used. Will it be good to declare a variable with value `source`?

Reply:
use something else than `source` which is bash command

Reply:
ah nevermind :D

Reply:
I am not exactly sure of the benefit here. Most of the time the source dir is changed in workingDir as
```
workingDir: $(workspaces.source.path)/source
```

Where I don't see a good way to declare that as a variable. I could change it just in the buildah tasks, for the 3 occurences, but that is not any big improvement. Should I go ahead and update if for the buildah task?

Reply:
I think declaring source directory as variable would be sufficient .. not sure about if there is a way to avoid repetitions in different task

Reply:
updated